src/printer-htmlparser2.js

```diff
@@ -103,7 +111,15 @@ function printAttributes(attribs) {
         if (attribs[name] === "") {
           return name;
         }
-        return concat([name, '="', attribs[name], '"']);
+
+        // FastPath requires the next node to be a property of the current one
+        node._currentAttribute = {
```

Comment:
Adding it to the ast in this way is reasonable to me by the way, if you move it to the parsing phase, I'll be happy with it.